public/themes/default/css/editor/notices.css

```diff
@@ -7,24 +9,29 @@ body {
     padding-right: 3em;
 
     font-family: Arial;
-    box-shadow: 2px 2px 10px 0px black;
+    box-shadow: 2px 2px 10px 0px var(--notice-light-box-shadow);
     border-radius: 0.5em;
     z-index: 10;
     opacity: 1;
     transition: opacity 0.5s linear;
 
     &.info {
-      background: #99ffaf;
-      text-shadow: 1px 1px 1px #ececec;
+      background: var(--notice-info-background);
+      color: var(--black);
```

Comment:
I'm not a big fan of this, because while it's useful to have both a color and background color, "black" has nothing to do with being a good contrasting color to "notice-info-background" (since we don't know what that variable actually resolves to).

So either we add color definitions but make them use their own named variables (and then _those_ can be a variable assignment to black, of course), or we leave these off for now.